node/pkg/governor/governor_test.go

```diff
@@ -1310,6 +1310,68 @@ func TestTestnetConfigIsValid(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestNumDaysForReleaseTimerReset(t *testing.T) {
+	ctx := context.Background()
+	gov, err := newChainGovernorForTest(ctx)
+
+	require.NoError(t, err)
+	assert.NotNil(t, gov)
```

Comment:
```suggestion
```
This should be tested elsewhere and doesn't explicitly need to exist. If you want to be super explicit and have extra verbose tests, you should also do a similar assertion for `tokenBridgeAddr` existing. I'd suggest don't.

Reply:
normal me would sort of assume that if there is no `err`, `gov` should be fine to use. Here, I actually just did what every other test in the file does. but happy to just remove.

Reply:
Your assumption is "the way" in go, and generally the correct one. The other tests are wrong as well :)